package.json

```
@@ -70,7 +70,7 @@
     ]
   },
   "engines": {
-    "node": ">=4.6.2"
+    "node": "^16 || ^14 || ^12"
```

This would cause the installation to break if folks are using Yarn on other versions of Node (such as Node 17). I'd recommend that we add our lowest target here instead.
```suggestion
    "node": ">=16"
```

---

makes sense to me. PR updated to `>=12` thank you for the review